autoload/OmniSharp/actions/members.vim

```diff
@@ -74,20 +74,20 @@ function! s:ComputeItemSignature(item) abort
   if textBeforeDisplayName !~# '^\(private\|internal\|protected\|public\)'
     let textBeforeDisplayName = a:item.Properties.accessibility . ' ' . textBeforeDisplayName
   endif
-  return ReduceToOneCharacter(textBeforeDisplayName) . a:item.DisplayName
+  return s:ReduceToOneCharacter(textBeforeDisplayName) . a:item.DisplayName
 endfunction
 
-let s:SingleCharacterSymbolByAccessModifier = {
- \ 'public': '+',
- \ 'private': '-',
- \ 'internal': '&',
- \ 'protected': '|'
-\}
 
-function! ReduceToOneCharacter(textBeforeDisplayName) abort
+function! s:ReduceToOneCharacter(textBeforeDisplayName) abort
+  let l:SingleCharacterSymbolByAccessModifier = {
+        \ 'public': '+',
+        \ 'private': '-',
+        \ 'internal': '&',
+        \ 'protected': '|'
+        \}
+
   let accessModifier = matchlist(a:textBeforeDisplayName, '\w\+')[0]
-  let accessModifierLen = len(accessModifier)
-  return s:SingleCharacterSymbolByAccessModifier[accessModifier] . a:textBeforeDisplayName[accessModifierLen:]
+  return l:SingleCharacterSymbolByAccessModifier[accessModifier] . a:textBeforeDisplayName[len(accessModifier):]
```

Comment:
Good move. I think you are right, that this was a vint false-positive, but it was an annoying one, it also shows up in my local environment.